packages/animations/ci_config.yaml

```diff
@@ -0,0 +1,2 @@
+release:
+  batch: false
```

Comment:
What's the reason for requiring every package to explicitly configure everything? I was envisioning this being more like a Dart test config file, where a package only needs a file if it needs non-default behavior.

Requiring every package to explicitly declare everything would make this a maintenance nightmare, because every time we add a new config option we would need to touch every single package.